.github/goreleaser.yaml

```diff
@@ -103,9 +65,6 @@ dockers:
   - image_templates:
       - "pomerium/pomerium:amd64-{{ .Tag }}"
     dockerfile: .github/Dockerfile-release
-    binaries:
```

Comment:
I assume we don't have to specify the explicit binary ids anymore since there is just the one now?